beekeeper-api/src/main/java/com/expediagroup/beekeeper/api/response/MetadataResponseConverter.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (C) 2019-2021 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.beekeeper.api.response;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+
+import com.expediagroup.beekeeper.core.model.HousekeepingMetadata;
+
+public final class MetadataResponseConverter {
+
+  private static HousekeepingMetadataResponse convertToHousekeepingMetadataResponse(
```

Comment:
Put this method below he public method as its used by that method. Gives better readability